Source/WebCore/rendering/RenderBlock.cpp

```diff
@@ -2033,6 +2033,19 @@ bool RenderBlock::isPointInOverflowControl(HitTestResult& result, const LayoutPo
 
 Node* RenderBlock::nodeForHitTest() const
 {
+    // If we're a backdrop pseudo-element, we should hit-test to the element that generated it.
+    // See webkit.org/b/233072 for more context.
```

Comment:
I’m not too fond of bug links for comments that will still be around years later. Do we really need to do it this way? Can we write just a few more words here that will elucidate.